plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/config.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import MISSING, ObjectConf
+
+
+@dataclass
+class EarlyStopConfig:
+
+    minimize: bool = MISSING
+
+    max_epochs_without_improvement: int = 10
+
+    # An improvement larger than epsilon is considered significant
+    epsilon: float = 0.00001
+
+
+@dataclass
+class ExperimentConfig:
+
+    # Experiment name
+    name: Optional[str] = None
+
+    # Name of metric to optimize or null if you only return one metric
+    objective_name: str = "objective"
```

Comment:
seems like a good candidate for Enum

Reply:
I am not sure if this should be enum. Looking at the [documentation](https://ax.dev/api/service.html#ax.service.ax_client.AxClient.create_experiment), it seems that the value can be any arbitrary string.

Reply:
Ah, I see.